ntbs-service/Pages/Notifications/NotificationsBase.cshtml.cs

```diff
@@ -16,24 +16,18 @@ public class NotifyError {
         public List<string> ErrorMessages { get; set; }
     }
 
-    public abstract class NotificationModelBase : ValidationModel
+    public abstract class NotificationEditModelBase : NotificationModelBase
     {
-        protected INotificationService service;
+        protected ValidationService validationService;
 
-        public NotificationModelBase(INotificationService service) 
+        public NotificationEditModelBase(INotificationService service) : base(service)
         {
-            this.service = service;
+            validationService = new ValidationService(this);
```

Comment:
This can't be injected as it needs the current page model - so this is tightly coupled by definition. It makes me feel a bit weird instantiating a service class like this though somehow (but I can't see how we couldn't in this case), hence my alternative suggestion to use extension methods - not sure what I prefer though

Reply:
Yes having thought about it more I do think this is fine :) I've just been indoctrinated by DI to the extent where I'm scared to use the word "new" haha